Gemfile.lock

```diff
@@ -27,6 +27,7 @@ GEM
     eventmachine (1.2.7)
     ffi (1.16.3)
     forwardable-extended (2.6.0)
+    google-protobuf (3.25.1-x86_64-darwin)
```

Comment:
Updates to this file should not be a part of this PR.

Reply:
Good catch, I just removed it